<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PracticeClass</class>
 <widget class="QMainWindow" name="PracticeClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1040</width>
    <height>672</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Practice</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QChartView" name="graphic">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>1021</width>
      <height>431</height>
     </rect>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="pte_data">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>460</y>
      <width>311</width>
      <height>171</height>
     </rect>
    </property>
    <property name="acceptDrops">
     <bool>false</bool>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QComboBox" name="combo_avg">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>460</y>
      <width>191</width>
      <height>22</height>
     </rect>
    </property>
    <property name="currentText">
     <string>As is</string>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <item>
     <property name="text">
      <string>As is</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Hourly</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>3 hour</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Per day</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Min per day</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Max per day</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="combo_graph_type">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>490</y>
      <width>191</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Linear</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Columnar</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Custom</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="clear_button">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>460</y>
      <width>41</width>
      <height>111</height>
     </rect>
    </property>
    <property name="text">
     <string>Clear</string>
    </property>
   </widget>
   <widget class="QComboBox" name="combo_params">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>490</y>
      <width>131</width>
      <height>22</height>
     </rect>
    </property>
    <property name="currentText">
     <string/>
    </property>
   </widget>
   <widget class="QProgressBar" name="progress_bar">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>610</y>
      <width>381</width>
      <height>21</height>
     </rect>
    </property>
    <property name="value">
     <number>100</number>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>570</y>
      <width>47</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Status:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_status">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>570</y>
      <width>321</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>OK</string>
    </property>
   </widget>
   <widget class="QPushButton" name="add_series_button">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>520</y>
      <width>331</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Add</string>
    </property>
   </widget>
   <widget class="QComboBox" name="combo_source">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>460</y>
      <width>131</width>
      <height>22</height>
     </rect>
    </property>
    <property name="currentText">
     <string/>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="pte_log">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>460</y>
      <width>311</width>
      <height>171</height>
     </rect>
    </property>
    <property name="acceptDrops">
     <bool>false</bool>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_data">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>435</y>
      <width>311</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Data</string>
    </property>
    <property name="textFormat">
     <enum>Qt::AutoText</enum>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>435</y>
      <width>311</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Log</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1040</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Data</string>
    </property>
    <addaction name="action_open_data"/>
    <addaction name="separator"/>
    <addaction name="action_download_data"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <action name="action_open_data">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="action_download_data">
   <property name="text">
    <string>Download</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QChartView</class>
   <extends>QGraphicsView</extends>
   <header>QtCharts</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="Practice.qrc"/>
 </resources>
 <connections/>
</ui>
